nutella/src/utils/axios/refresh.ts

```diff
@@ -6,17 +6,43 @@ export const request = axios.create({
   timeout: 100000,
 });
 
+const getDateWithAddHour = (hour: number) => {
+  const date = new Date();
+  date.setHours(date.getHours() + hour);
+  return date;
+};
+
 const refresh = async (
   config: AxiosRequestConfig
 ): Promise<AxiosRequestConfig> => {
   const expireAt = localStorage.getItem("expire_at");
   const accessToken = localStorage.getItem("access_token");
   const refreshToken = localStorage.getItem("refresh_token");
+  if (!refreshToken || !expireAt) {
+    window.location.href = "/";
+    localStorage.removeItem("expire_at");
+    localStorage.removeItem("access_token");
+    localStorage.removeItem("refresh_token");
+
+    return config;
+  }
+  /* 이부분링크에code를담아서보내야하는데방법을잘모르겠어...!
+   const uri = `auth/callback-google?code=${code}`;
+  try {
+    const { accessToken, refreshToken } = (await request.post(uri))
```

Comment:
코드를 담으려면 uri 자체에 담지 말고
```suggestion
    const uri = "auth/callback-google";
    const { accessToken, refreshToken } = (await request.post(uri, null, { params:{ code: code } } ))
```
이렇게 params 안에 담으면
자동으로 
```
auth/callback-google?code=yourcode
```
로 바꿔줍니다